content/docs/command-reference/import-url.md

```diff
@@ -135,16 +135,19 @@ produces a regular stage in `dvc.yaml`.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create the import `.dvc` file but don't download `url` or get
-  checksums (assumes that the data source is valid). This is useful if you need
-  to define the project imports quickly, and import the data later (use
-  `dvc update` to finish the operation(s)).
-
-- `--no-download` - create the import `.dvc` with data checksums but without
-  downloading the associated data. This is useful if you need track changes in
-  remote data but do not (yet) need to download data to the local workspace.
-  Data can be later downloaded using `dvc pull`, but will fail if the data
-  matching the checksum is no longer available.
+- `--no-exec` - create the import `.dvc` file without accessing `url` (assumes
+  that the data source is valid). This is useful if you need to define the
+  project imports quickly, and import the data later (use `dvc update` to finish
+  the operation(s)).
+
+- `--no-download` - create the import `.dvc` file including
+  [hash values](/doc/user-guide/project-structure/dvc-files#output-entries) for
```

Comment:
Wait. Should this link to https://dvc.org/doc/user-guide/project-structure/dvc-files#dependency-entries, actually? What hash values are calculated here (for `deps` or for `outs`)?

Reply:
Indeed, that's an error on me: I meant to link to the deps table, the link you posted. The included hash values are etag/checksum for the deps section.